docs/python-configuration.md

```diff
@@ -83,6 +208,7 @@ scope:
   instrumentation:
     http:
       headers:
+        - Authorization
```

Comment:
this is a bad example for python as we will still redact the header

Reply:
Deleted the line, should you add an issue in the agent?